spark/src/test/scala/org/apache/spark/sql/delta/schema/InvariantEnforcementSuite.scala

```diff
@@ -407,14 +407,9 @@ class InvariantEnforcementSuite extends QueryTest
         val newMetadata = txn.metadata.copy(
           configuration = txn.metadata.configuration +
             ("delta.constraints.mychk" -> "valueA < valueB"))
-        assert(txn.protocol.minWriterVersion === writerVersion)
         txn.commit(Seq(newMetadata), DeltaOperations.ManualUpdate)
-        val upVersion = if (TableFeatureProtocolUtils.supportsWriterFeatures(writerVersion)) {
-          TableFeatureProtocolUtils.TABLE_FEATURES_MIN_WRITER_VERSION
-        } else {
-          CheckConstraintsTableFeature.minWriterVersion
-        }
-        assert(table.deltaLog.unsafeVolatileSnapshot.protocol.minWriterVersion === upVersion)
+        assert(table.deltaLog.unsafeVolatileSnapshot.protocol.minWriterVersion ===
```

Comment:
```suggestion
        assert(table.deltaLog.update().protocol.minWriterVersion ===
```
While we are at it^^